make/photon/prepare/utils/portal.py

```diff
@@ -19,3 +21,11 @@ def prepare_portal(config_dict):
         gid=DEFAULT_GID,
         strong_ssl_ciphers=config_dict['strong_ssl_ciphers']
         )
+
+    # Render Portal setting config
+    if config_dict['look_and_feel']:
```

Comment:
Did you mean?

> If config_dict has something stored in `'look_and_feel'` key: then render the jinja remplate`

```suggestion
    if 'look_and_feel' in config_dict:
```
or:

```suggestion
    if config_dict.get('look_and_feel', False):
```